src/actions/__tests__/Button.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { render, cleanup, fireEvent } from "@testing-library/react"
+import Button from "../Button"
+
+afterEach(cleanup)
+
+describe("<Button>", () => {
+  it("displays a clickable button", () => {
+    let wasClicked = false
+    const content = "Button Label"
+    const { getByText, container } = render(
+      <Button id="test-button" className="test-class" onClick={() => (wasClicked = true)}>
+        {content}
+      </Button>
+    )
+    expect(getByText(content)).toBeInTheDocument()
+    expect(
+      container.querySelector("#test-button.test-class[data-variant='primary']")
+    ).not.toBeNull()
+
+    fireEvent.click(getByText(content))
+
+    expect(wasClicked).toBeTruthy()
+  })
+
+  it("displays an anchor with an href", () => {
+    const content = "Button Label"
+    const { getByRole, container } = render(<Button href="/my-page">{content}</Button>)
+
+    expect(getByRole("link", { name: content })).toHaveAttribute(
+      "href",
+      "/my-page"
+    )
+    expect(container.querySelector(".seeds-icon")).toBeNull()
```

Comment:
Do you think it would be possible to check for an SVG here? I think that might be slightly less brittle